x/evm/types/chain_config.go

```diff
@@ -50,8 +52,8 @@ func (cc ChainConfig) EthereumConfig(chainID *big.Int) *params.ChainConfig {
 		ArrowGlacierBlock:       getBlockValue(cc.ArrowGlacierBlock),
 		GrayGlacierBlock:        getBlockValue(cc.GrayGlacierBlock),
 		MergeNetsplitBlock:      getBlockValue(cc.MergeNetsplitBlock),
-		ShanghaiBlock:           getBlockValue(cc.ShanghaiBlock),
-		CancunBlock:             getBlockValue(cc.CancunBlock),
+		ShanghaiTime:            &shangHaiBlock,
```

Comment:
@yihuang  since ethereum change to [timetamp](https://github.com/ethereum/go-ethereum/blob/v1.11.0/params/config.go#L435), do we need change [proto](https://github.com/crypto-org-chain/ethermint/blob/2886d0f8883aa1d632c40cdd7708322cdddc8999/x/evm/types/evm.pb.go#L159)?